clang/include/clang/Basic/Attr.td

```diff
@@ -3990,6 +3990,13 @@ def LocksExcluded : InheritableAttr {
   let Documentation = [Undocumented];
 }
 
+def ReentrantCapability : InheritableAttr {
+  let Spellings = [Clang<"reentrant_capability">];
+  let Subjects = SubjectList<[Record, TypedefName]>;
+  let Documentation = [Undocumented];
```

Comment:
We generally don't want new, undocumented attributes. However, I think this is fine because we have dedicated documentation for this elsewhere which is being updated, and this matches the same lack of documentation as the rest of the thread safety attributes.

Reply:
Ack. Yes, style follows the rest of these attributes.

Reply:
IMO we could at least add a link to the other documentation. Just saying that it's undocumented in the place where all other attributes are documented is really not helpful.